subprojects/core-api/src/main/java/org/gradle/internal/metaobject/NamedReference.java

```diff
@@ -22,7 +22,7 @@
 /**
  * Provides dynamic access to properties of some object.
  */
-public interface PropertyAccess {
+public interface NamedReference {
```

Comment:
❓  This looks like an undesired rename, isn't it?

Reply:
No, we explicitly agreed to do it.

Reply:
What Paul meant is that this `PropertyAccess` is unrelated to DCL.

Reply:
Fixed now.